libs/ledgerjs/packages/hw-app-celo/tests/Celo.test.ts

```diff
@@ -0,0 +1,8 @@
+import { openTransportReplayer, RecordStore } from "@ledgerhq/hw-transport-mocker";
+import Celo from "../src/Celo";
+
+test("Celo init", async () => {
+  const transport = await openTransportReplayer(RecordStore.fromString(""));
+  const dot = new Celo(transport);
+  expect(dot).not.toBe(undefined);
```

Comment:
`not.toBe(undefined)` is such a poor assertion. What about
```suggestion
  const celo = new Celo(transport);
  expect(celo).toBeInstanceOf(Celo);
```